trade-finance-manager-api/src/v1/controllers/user/user.controller.js

```diff
@@ -106,6 +107,19 @@ exports.update = async (_id, update, auditDetails, callback) => {
   });
 };
 
+/**
+ * Creates or updates a TFM user from an Entra user.
```

Comment:
"SSO" instead "Entra" or maybe "Entra SSO"?

Reply:
Am going to add this is in the SSO login process (the reason to stick to entra is that this is more in line with GIFT.)

I do think you have a great point about making things easily identifiable as SSO though, so I'll update comments to suite!